modules/select2_publish/select2_publish.libraries.yml

```diff
@@ -0,0 +1,8 @@
+select2.publish:
+  css:
+    component:
+      css/select2.publish.css: {}
+  js:
+    js/select2.publish.js: { weight: -1 }
```

Comment:
This part should be documented next to `$(this).trigger('select2-init');` - so that someone who is going to use same API, take care about that. Otherwise it will not work.

I have checked in what order behaviours are loaded and they are using `Object.keys()` - what returns keys of object in order they are added to object, but it's not guaranteed.